### PR TITLE
Add notice about distro package changes to 5.5.2 release notes

### PR DESCRIPTION
**Pull request by mfussenegger**

### CODE REVIEW

File: docs/appendices/release-notes/5.5.2.rst [hunk 1]
```diff
@@ -46,6 +46,39 @@ See the :ref:`version_5.5.0` release notes for a full list of changes in the
...
+    - ``bin``, ``jdk`` and ``lib`` are installed into ``/usr/share/crate``. In the RPM
+      package this used to be in ``/opt/crate``.
+
+    - The home directory of the crate user is ``/usr/share/crate``
+
+    - changes, notice, license are in ``/usr/share/doc/crate``
+
+    - service file is in ``/usr/lib/systemd/system``
+
+    - The ``crate.yml`` configuration file is in ``/etc/crate/``
```
Comment (seut) [inline docs/appendices/release-notes/5.5.2.rst:67]: I guess this didn't change even for RPM (because you added an explicit comment above for RPM)?

**Review by seut — APPROVED** [review]
lgtm otherwise

File: docs/appendices/release-notes/5.5.2.rst (hunk 1 above)
Comment (mfussenegger) [reply docs/appendices/release-notes/5.5.2.rst:67]: Yes it didn't change, but thought I'd add it to give a complete overview. Should I remove it?

Comment (seut) [reply docs/appendices/release-notes/5.5.2.rst:67]: 👍 I like the overview

**Pull request merged**